src/main/resources/swagger.api/schemas/entityTypeColumn.json

```diff
@@ -33,10 +33,21 @@
         "essential": {
           "description": "Indicates if a column is crucial to the definition of an entity type",
           "type": "boolean"
+        },
+        "joinsTo": {
+          "description": "A set of columns which this column may be joined to",
+          "type": "array",
+          "items": {
+            "$ref": "./field-joins/join.json"
+          }
+        },
+        "originalEntityTypeId": {
```

Comment:
Used by the flattening service to keep track of which column came from where, to ensure compatibility